aws-private-existing-vpc/README.md

```diff
@@ -0,0 +1,33 @@
+# MageAI in AWS
+This module deploys MageAI with the following assumptions:
+* MageAI will be added to an existing VPC
+* Deployment should be to private subnets
+* The control database should not be publicly accessible
+* Your deployment has a VPN for accessing resources on the private subnet.
+
+## Pre-Deployment
+Before deployment, you must have the following installed:
+* [Terragrunt](https://terragrunt.gruntwork.io/)
```

Comment:
It can also be run without Terragrunt, right?

Reply:
It can now I've added back the `env_vars.json`.